fiovb/ta/entry.c

```diff
@@ -160,6 +162,66 @@ static TEE_Result delete_value(char *name, size_t name_sz)
 	return res;
 }
 
+static bool is_rollback_protected(void)
+{
+	TEE_Result res = TEE_SUCCESS;
+	uint32_t value_sz = MAX_SIMPLE_VALUE_SIZE;
+	char value[MAX_SIMPLE_VALUE_SIZE];
+	uint32_t count;
+
+	res = read_value(ROLLBACK_PROT, strlen(ROLLBACK_PROT) + 1,
+			value, value_sz, &count);
+	if (res == TEE_SUCCESS) {
+		IMSG("Found %s, rollback protection is enabled",
```

Comment:
Should we make this DMSG instead? Same for other IMSG, I think we should just print when there is an error.

Reply:
I think it's better to have IMSG, no notify explicitly that rollback protection is enabled
for an error `EMSG` is usually used

Reply:
fixed as we discussed